nes/search/__init__.py

```
@@ -0,0 +1,29 @@
+from nes.search.indexer import EntityIndexer
+from nes.search.models import SearchHit, SearchQuery, SearchResults
+
+__all__ = [
+    "SearchBackend",
+    "SearchQuery",
+    "SearchHit",
+    "SearchResults",
+    "EntityDocumentBuilder",
+    "EntityIndexer",
+    "get_search_backend",
+]
```

Ruff (0.15.18)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)